src/btree/bt_split.c

```diff
@@ -1489,6 +1489,7 @@ __split_multi_inmem(WT_SESSION_IMPL *session, WT_PAGE *orig, WT_MULTI *multi, WT
             for (; tmp != NULL && tmp != supd->onpage_upd; tmp = tmp->next)
                 WT_ASSERT(session, tmp == supd->onpage_tombstone || tmp->txnid == WT_TXN_ABORTED);
 #endif
+            /* Discard updates/tombstone after prev_onpage */
```

Comment:
This comment was likely from investigating the update restore path. It's actually a good comment to have regardless as that code path isn't well commented. Could you add a fullstop to the end?

Reply:
Done